config/validator.go

```diff
@@ -131,6 +134,16 @@ func mergeErrors(maps ...map[string][]error) (output map[string][]error) {
 	return output
 }
 
+// prepareErrors
+func prepareErrors(errList []error, validatorName string) (validationErrors map[string][]error, valid bool) {
+	if valid = len(errList) < 1; !valid {
+		errorsList := make(map[string][]error)
+		errorsList[validatorName] = errList
+		validationErrors = mergeErrors(validationErrors, errorsList)
```

Comment:
Shouldn't this instruction be outside of the condition? Because if there are no errors, `validationErrors` will be nil? Right?

Reply:
It's no problem - all higher order functions using `valid` returned arg. in this case.
In below example value returned by `validationErrors` isn't usable, when `valid == True`.

Reply:
k